### DISCUSSION TITLE
Exeception Io - StopWatch

### DISCUSSION THREAD

**Discussion by CleitonTito** [balta-io/2801#49] [Perguntas e Respostas] [answered]
Boa tarde,

Retornei as aulas e estou fazendo o projeto do Stopwatch, porem ao inserir no meu código o Console.Clear() a fim de limpar a tela o vs code não me traz nenhuma observação, quando aperto o F5 para debugar e compilar me aparece IOException.
Como posso solucionar  ou tratar essa execeção?
Deixei um print em anexo e vou deixar o códgio aqui:

using System;
using System.Threading;

namespace StopWhatch
{
    class Program
    {
        static void Main(string[] args)
        {
            Start();
        }

        static void Start()
        {
            int time = 10;
            int currentTime = 0;

            while (currentTime != time)
            {
                Console.Clear();
                currentTime++;
                Console.WriteLine(currentTime);
                Thread.Sleep(1000);

            [}](![image](https://user-images.githubusercontent.com/81769859/154720482-01c20167-18fb-462d-a7ed-b4c924b6be49.png))

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @CleitonTito, como vai?

Qual erro ocorre?

**Reply by CleitonTito** [reply]
Boa tarde André,

Exception has occurred: CLR/System.IO.IOException

Ocorreu uma exceção sem tratamento do tipo "System.IO.Exception" em System.Console.dll: 'Identificador inválido.,

**Reply by andrebaltieri** [reply]
Tenta executar um `dotnet clean` e depois um `dotnet run` novamente! Pode ser algo do cache... se puder enviar um print da tela toda, seria legal, só pelo erro não consegui identificar.

**Reply by CleitonTito** [reply]
Certo, segue a imagem:

![Erro Execcao v1](https://user-images.githubusercontent.com/81769859/154989166-61638e0e-5976-4b4c-ae23-fe3bf9c4ad4c.PNG)

**Reply by andrebaltieri** [reply]
Nunca vi este erro antes. Pelo que pesquisei, algumas pessoas tiveram isto, e estava relacionada ao processo sendo executado (Usuário no Windows). Você consegue remover ela e seguir para não ficar travado?

Aparentemente, seu código está OK!

**Reply by CleitonTito** [reply]
Blz André,

Obrigado por todo o apoio, vou dar continuidade nas aulas!
